requirements-ci.txt

```diff
@@ -1,6 +1,6 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
+# by the following command:
```

Comment:
Why renaming the default content of the autogenerated files of [`pip-tools`](https://github.com/jazzband/pip-tools)?